modin/numpy/arr.py

```diff
@@ -294,7 +282,20 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
             args += [
                 input._query_compiler if hasattr(input, "_query_compiler") else input
             ]
-        return array(_query_compiler=new_ufunc(*args, **kwargs), _ndim=out_ndim)
+        out_kwarg = kwargs.get("out", None)
+        if out_kwarg is not None:
+            out_kwarg = out_kwarg[0]
```

Comment:
Why?

Reply:
If the `out_kwarg` is a modin.numpy.array, getting it from kwargs returns a 1-length tuple containing the array. I'll add a comment clarifying this as well!